pipelines/build/common/build_base_file.groovy

```diff
@@ -828,7 +828,7 @@ class Builder implements Serializable {
                             // Pass down DEFAULTS_JSON
                             buildJobParams.add(['$class': 'TextParameterValue', name: 'DEFAULTS_JSON', value: JsonOutput.prettyPrint(JsonOutput.toJson(DEFAULTS_JSON)) ])
 
-                            def downstreamJob = context.build job: downstreamJobName, propagate: false, parameters: buildJobParams
+                            def downstreamJob = context.build job: downstreamJobName, propagate: true, parameters: buildJobParams
```

Comment:
This one needs to stay "false", as the PARAM.propagateFailures check in the if (downstreamJob.getResult() == 'SUCCESS') ... else...
handles it